app/javascript/components/payment-error-button.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="payment-error-button">
+    <modal name="apology-modal">
+      <div class="invoice__apology-modal-text">
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
```

Comment:
Expected indentation of 8 spaces but found 10 spaces                       vue/html-indent